plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version4/Products/class-wc-rest-products-v4-controller-tests.php

```
@@ -0,0 +1,1994 @@
+	 */
+	public function test_products_filter_with_valid_invalid_exclude_status() {
+		$request = new WP_REST_Request( 'GET', '/wc/v4/products' );
+		$request->set_query_params(
+			array(
+				'exclude_status' => array( ProductStatus::DRAFT, 'invalid_status' ),
+			)
+		);
+
+		$response = $this->server->dispatch( $request );
+		$this->assertEquals( 400, $response->get_status() );
+		$data = $response->get_data();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (Unused Code Rules)

(UnusedLocalVariable)